domains/ratmilk.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "MilkedRat",
+        "email": "keithpumpkin@gmail.com"
+    },
+    "record": {
+        "URL": "https://youtu.be/dqw4w9wgxcq"
```

Comment:
The video you pointed the domain to is not avaible (private or deleted ig)
"This video isn't available anymore"

Reply:
(btw if you want the rickroll, the video ID is `dQw4w9WgXcQ`, you should keep the uppercase ones)

Reply:
thats my bad, i copied the wrong link to go to the rickroll, I meant to copy it from the address bar but I did an oopsie instead